README.md

```diff
@@ -141,8 +141,10 @@ For the default values look at the `Dockerfile`.
   * `X-Forwarded-Proto` RequestHeader (in default site)
 * APACHE_TIMEOUT
   * `Timeout` (in Apache configuration)
+* PROXY_PRESERVE_HOST
+  * `ProxyPreserveHost` default `on` (in default site)
```

Comment:
Move the default into the line above, please. E.g.
```markdown
* PROXY_PRESERVE_HOST (Default: `on`)
  * `ProxyPreserveHost` (in default site)
```

It should match the layout of the [Configuration](https://github.com/vshn/modsecurity-docker#configuration) section above.